src/transforms/coerce.jl

```diff
@@ -0,0 +1,43 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Coerce(pairs, tight=false, verbosity=1)
+
+Return a copy of the table, ensuring that the scientific types of the columns match the new specification.
+
+This transform wraps the ScientificTypes.coerce function. Please see their docstring for more details.
+
+```julia
+Coerce(:col1 => Continuous, :col2 => Count)
+```
+"""
+struct Coerce{P} <: Transform
+  pairs::P
+  tight::Bool
+  verbosity::Int
+end
+
+Coerce(pair...; tight=false, verbosity=1) = Coerce(pair, tight, verbosity)
```

Comment:
```suggestion
Coerce(pair::Pair{Symbol,<:Type}...; tight=false, verbosity=1) = 
  Coerce(pair, tight, verbosity)
```
Type annotations are very useful for showing better error messages.